[WIP] Fix remaining remote backend references (#30235)
hashicorp/terraform: website/docs/cli/import/index.mdx

While this may seem tedious, it still gives Terraform users an avenue for
importing existing resources.

## Terraform Cloud

> When you use Terraform on the command line with Terraform Cloud, many commands (e.g., `apply`) run inside your Terraform Cloud environment. However, the `import` command runs locally, so it will not have access to information from Terraform Cloud. To fix this, you may need to set local variables equivalent to any remote workspace variables in Terraform Cloud.

Reviewer: Well, that doesn't quite land you in a "fixed" state where everything works; it's more just a required hunk of extra work to do the thing. How about: 

```suggestion
When you use Terraform on the command line with Terraform Cloud, many commands (e.g., `apply`) run inside your Terraform Cloud environment. However, the `import` command runs locally, so it will not have access to information from Terraform Cloud. To successfully perform an import, you may need to set local variables equivalent to any remote workspace variables in Terraform Cloud.
```
